playwright/fixtures/common.ts

```diff
@@ -0,0 +1,100 @@
+import { _electron as electron } from "playwright";
+import { test as base, expect } from "@playwright/test";
+import * as fs from "fs";
+import * as path from "path";
+import * as crypto from 'crypto';
+
+export function generateUUID(): string {
+  return crypto.randomBytes(16).toString('hex');
+}
+
+type TestFixtures = {
+  lang: String;
+  theme: String;
+  userdata: String;
+  env: Object;
+  page: any;
```

Comment:
Can this be changed to `Page`?

Reply:
we could use `Page` from `@playwright/test`

Reply:
Thank you guys!